examples/multiple_test_pages/tests.js

```diff
@@ -0,0 +1,11 @@
+for (var i = 0; i < 200; i++) {
+  if (window.location.search === '?batch=1') {
```

Comment:
Going forward, we could likely define a convention (batch=1&batch-size=10) or something, then push the popular runners to support this.